apps/govern/components/Docs/Docs.tsx

```diff
@@ -0,0 +1,199 @@
+import { Divider, Flex, Tag, Typography } from 'antd';
+import Image from 'next/image';
+import styled from 'styled-components';
+
+import { COLOR } from 'libs/ui-theme/src';
+
+const { Paragraph, Title, Text } = Typography;
+
+const Hero = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: 1312px;
+  margin: 0 auto;
+`;
+
+const ContentWrapper = styled.div`
+  padding: 16px 0 40px;
+  border-top: 1px solid #d7ddea;
+  border-bottom: 1px solid #d7ddea;
+  background-color: ${COLOR.WHITE};
+  display: flex;
+  flex-direction: column;
+  margin: 0 -40px -40px -40px;
```

Comment:
our main layout has paddings and grey background, added negative margins here so these sections can have white background